.github/ISSUE_TEMPLATE/cxx20-feature.md

```diff
@@ -0,0 +1,24 @@
+---
+name: cxx20 Feature
+about: For STL maintainers only
+title: PAPER_NUMBER PAPER_TITLE
+labels: cxx20
+assignees: ''
+
+---
+
+**NOTE**
+This issue template is for STL maintainers only.
+Please use the Bug Report issue template instead.
+
+
+[PAPER_NUMBER](https://wg21.link/PAPER_NUMBER) PAPER_TITLE
+
+Note: If this link appears to be broken, don't worry! Papers are often
+revised during a WG21 meeting before the final vote. When that happens,
+there's a delay of a few weeks between the end of the meeting
+and the publication of the papers. When the papers are available,
+the https://wg21.link redirector will start working.

```

Comment:
I've noticed that wg21.link redirects to isocpp.org for papers submitted using the new automated system even before the mailing goes out (e.g., https://wg21.link/p2091r0). I don't know that this fact merits mention here, but FYI.

Reply:
Yep, I've noticed that too. I think we don't need to mention the complexity - the note begins with "If this link appears to be broken" and nobody's going to complain if it works.